src/app-layer-htp-file.c

```diff
@@ -150,6 +150,107 @@ int HTPFileOpen(HtpState *s, const uint8_t *filename, uint16_t filename_len,
     SCReturnInt(retval);
 }
 
+/**
+ * Performs parsing of the content-range value
+ *
+ * @param[in] rawvalue
+ * @param[out] range
+ *
+ * @return HTP_OK on success, HTP_ERROR on failure.
+ */
+int HTPParseContentRange(bstr * rawvalue, HtpContentRange_t *range) {
+    unsigned char *data = bstr_ptr(rawvalue);
+    size_t len = bstr_len(rawvalue);
+    size_t pos = 0;
+    size_t last_pos;
+
+    // skip spaces and units
+    while (pos < len && data[pos] == ' ') pos++;
```

Comment:
style: newline after the while condition
```c
while (pos < len && data[pos] == ' ')
    pos++;
```

Reply:
ok